hack/machineset.sh

```diff
@@ -6,7 +6,8 @@
 # USAGE
 #    machineset.sh
 # OPTIONS
-#    $1      Action       (Optional) apply/delete the MachineSet
+#    $1    WIN_SRV   (Optional) Windows Server version of VMs associated with the resulting MachineSet. Defaults to 2022
+#    $2    Action    (Optional) apply/delete the MachineSet
```

Comment:
This is a breaking change.
Given Windows Server option defaults to 2022, consider keeping the action as the first option ($1), and just extend with the Windows Server option.

Reply:
That wouldn't quite work since `Action` is optional. Reworked it in a breaking but hopefully better manner :slightly_smiling_face: